tests/classification/test_auc.py

```diff
@@ -76,3 +78,12 @@ def test_auc_functional(self, x, y):
 def test_auc(x, y, expected):
     # Test Area Under Curve (AUC) computation
     assert auc(tensor(x), tensor(y), reorder=True) == expected
+
+
+@pytest.mark.parametrize("nb", (None, 5, 15))
+def test_stable_1d_sort(nb):
```

Comment:
in your test, you are actually not testing stability at all, you are just testing if the sort is working at all. Can you try seeing if sorting an array of 8 zeros (and other equal values) produces indices=[0,1,2,3,..7] (so no changes are made by sort, meaning it's stable)